MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+recursive-include openfisca_web_api/i18n *.mo
+include *.md
+include LICENSE
+include *.ini
```

Comment:
So @pndsagora could you retry with the latest version of setuptools (`34.2.0` just now)?

If it doesn't work can you copy-paste:
- the build command with its output
- the output of `pip list`

thanks!